src/databricks/labs/ucx/assessment/workflows.py

```diff
@@ -183,17 +183,21 @@ def crawl_groups(self, ctx: RuntimeContext):
         """Scans all groups for the local group migration scope"""
         ctx.group_manager.snapshot()
 
+    @job_task
+    def assess_workflows(self, ctx: RuntimeContext):
+        """Scans all jobs for migration issues in notebooks jobs.
+
+        Also, stores direct filesystem accesses for display in the migration dashboard.
```

Comment:
stores tables as well

Reply:
I am working on a separate PR that updates the docs, separating this out for now